docs/source/user-guide/install/linux.rst

```diff
@@ -44,7 +44,13 @@ Using with fish shell
 
 To use conda with fish shell, run the following in your terminal:
 
-  ``conda init fish``
+ #. Add conda binary to $PATH, if not yet added::
+ 
+      fish_add_path path/to/bin
```

Comment:
```suggestion
      fish_add_path path/to/condabin
```

It's best to add only the `condabin` directory, which is designed to contain only conda binaries, and not other stuff that might be installed in `base` (e.g. `python`).